<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QMainWindow" name="home">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/resources/img/bg-home.jpg&quot;);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainframe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>11</y>
      <width>811</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainframe {
	background: rgba(255,255, 255,0.5);
	border-radius: 5;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: rgba(255,255,255, 0.7);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="newAccount">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: green;
border-radius: 5;
color: white;
font-size: 15px;
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Create new</string>
     </property>
    </widget>
    <widget class="QTableView" name="allUsers">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>711</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  white;
border: none;
border-radius: 10;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: rgba(255,255,255, 0.4);</string>
     </property>
     <property name="placeholderText">
      <string> Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logout">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>40</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: red;
border-radius: 5;
color: white;
font-size: 15px;
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <zorder>newAccount</zorder>
    <zorder>allUsers</zorder>
    <zorder>searchButton</zorder>
    <zorder>search</zorder>
    <zorder>logout</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
